tests/lib/rules/webpack/no-unnamed-dynamic-imports.js

```diff
@@ -1,91 +1,84 @@
 const {RuleTester} = require('eslint');
 const rule = require('../../../../lib/rules/webpack/no-unnamed-dynamic-imports');
 
-const ruleTester = new RuleTester();
+const ruleTester = new RuleTester({
+  parser: require.resolve('babel-eslint'),
+});
 
 const CHUNK_NAME_REQUIRED =
   'imports should have a webpackChunkName (https://webpack.js.org/api/module-methods/#import-)';
 
-const validExamples = [
-  {
-    code: `
-      function foo() {
-        return import(/* webpackChunkName: "bar" */ 'bar');
-      }
-    `,
-    parser: 'babel-eslint',
-  },
-  {
-    code: `
-      function foo() {
-        return import(/* webpackChunkName: 'bar' */ 'bar');
-      }
-    `,
-    parser: 'babel-eslint',
-  },
-  {
-    code: `
+ruleTester.run('webpack/no-unnamed-dynamic-imports', rule, {
+  valid: [
+    {
+      code: `
+        function foo() {
+          return import(/* webpackChunkName: "bar" */ 'bar');
+        }
+      `,
+    },
+    {
+      code: `
+        function foo() {
+          return import(/* webpackChunkName: 'bar' */ 'bar');
+        }
+      `,
+    },
+    {
+      code: `
+        async function foo() {
+          await import(/* webpackChunkName: 'bar' */ 'bar');
+        }
+      `,
+    },
+    {
+      code: `System.import(/* webpackChunkName: "bar" */ 'bar');`,
+    },
+    {
+      code: `
+        System.import(
+          /* webpackMode: "lazy" */
+          /* webpackChunkName: "bar" */ 'bar'
+        );
+      `,
+    },
+    {
+      code: `System.imported('foo');`,
+    },
+    {
+      code: `System.other('foo');`,
+    },
+    {
+      code: `
       async function foo() {
-        await import(/* webpackChunkName: 'bar' */ 'bar');
+        return System.import<{default: any}>(
+          /* webpackChunkName: "bar" */ './bar',
+        ).then(bar => bar);
       }
-    `,
-    parser: 'babel-eslint',
-  },
-  {
-    code: `System.import(/* webpackChunkName: "bar" */ 'bar');`,
-  },
-  {
-    code: `
-      System.import(
-        /* webpackMode: "lazy" */
-        /* webpackChunkName: "bar" */ 'bar'
-      );
-    `,
-  },
-  {
-    code: `System.imported('foo');`,
-  },
-  {
-    code: `System.other('foo');`,
-  },
-  {
-    code: `
-    async function foo() {
-      return System.import<{default: any}>(
-        /* webpackChunkName: "bar" */ './bar',
-      ).then(bar => bar);
-    }
-    `,
-    parser: 'typescript-eslint-parser',
-  },
-];
-
-const invalidExamples = [
```

Comment:
only changed for consistency with the rest of this repo: we do not ever store the examples in a variable before passing them into the ruleTester.